### DISCUSSION TITLE
Exception: Error: 404 Not Found ：invalid_request_error

### DISCUSSION THREAD

**Discussion by seanhuangjf** [acheong08/ChatGPT#1000] [Q&A]
raise Exception(
Exception: Error: 404 Not Found {
  "error": {
    "message": "Invalid URL (POST /v1/chat/completions)",
    "type": "invalid_request_error",
    "param": null,
    "code": null
  }

**Comment by acheong08** [comment]
Temporary error from OpenAI I think
